### ISSUE TITLE
RStudio *does* support ligatures now

### ISSUE THREAD

**Issue by msberends** [tonsky/FiraCode#496]
On [the Wiki](https://github.com/tonsky/FiraCode/wiki/RStudio-instructions): 
> There's no straightforward way to enable font ligatures in RStudio (...)

Well, now there is: the latest version supports it: https://blog.rstudio.com/2017/09/13/rstudio-v1.1---the-little-things/#ligature-support

They even refer to this very GitHub page:

_In the last few years we’ve seen an uptick in fonts designed specifically for code. [Many of these use “ligatures”](https://www.hanselman.com/blog/MonospacedProgrammingFontsWithLigatures.aspx), which combine two or more individual characters into a single typographical glyph (see [Fira Code](https://github.com/tonsky/FiraCode) for examples). RStudio Server and RStudio on MacOS have always supported these natively; in RStudio 1.1 we added support for them on Windows and Linux, too._

AND have a nice way of showing:
![image](https://user-images.githubusercontent.com/31037261/30915248-7b1aaa2e-a396-11e7-85d6-a3715f3f0882.png)

Bottom line: please update Wiki :smiley:

**Comment by tonsky** [comment]
Thanks! Removed instructions from wiki
